client/src/App.js

```diff
@@ -68,16 +54,32 @@ const App = props => {
     localStorage.setItem("currentUser", JSON.stringify(loggedInUser));
   };
 
+  //TODO: This doesn't seem like it's getting used anymore. Don't see token in local storage. Check on authorization flow to see if token is still needed.
+  const setTokenInHeaders = () => {
+    axios.interceptors.request.use(
+      config => {
+        let token = localStorage.getItem("token");
+        if (token) {
+          config.headers["Authorization"] = `Bearer ${token}`;
+        }
+        return config;
+      },
+      error => Promise.reject(error)
+    );
+  };
+
+  setTokenInHeaders();
+

```

Comment:
This looks like something unrelated to the ticket. Can you move it back to above the App declaration where it used to be?
It looks like something used for some manual testing purpose.